scripts/servers.py

```diff
@@ -722,3 +722,50 @@ def managed_webdriverio_server(
             yield proc
     finally:
         del os.environ['MOBILE']
+
+
+@contextlib.contextmanager
+def managed_acceptance_tests_server(
+    suite_name: str = 'full',
+    sharding_instances: int = 1,
+    chrome_version: Optional[str] = None,
+    stdout: int = subprocess.PIPE,
+) -> Iterator[psutil.Process]:
+    """Returns context manager to start/stop the Acceptance server gracefully.
+
+    Args:
+        suite_name: str. The suite name whose tests should be run. If the value
+            is `full`, all tests will run.
+        sharding_instances: int. How many sharding instances to be running.
+        chrome_version: str|None. The version of Google Chrome to run the tests
+            on. If None, then the currently-installed version of Google Chrome
+            is used instead.
+        stdout: int. This parameter specifies the executed program's standard
+            output file handle.
+
+    Yields:
+        psutil.Process. The webdriverio process.
+
+    Raises:
+        ValueError. Number of sharding instances are less than 0.
+    """
+    if sharding_instances <= 0:
+        raise ValueError('Sharding instance should be larger than 0')
+
+    if chrome_version is None:
+        chrome_version = get_chrome_version()
+
+    acceptance_tests_args = [common.NODE_BIN_PATH, suite_name]
```

Comment:
The path is still only node and suite, which is confusing.

Reply:
The jasmine PR #17401 needs to go before this. Because i have a few ways of carrying out the tests for eg. whether we want to run each suite with a seperate runner so there is no need for restarting the servers again. Or we can just maintain one suite and maintain common superusers (superadmin, blogadmin, etc. ) to use in each of the tests. That's why i am currently unsure about updating it.

Reply:
Run each suite with a separate runner for now. That's the quickest way to get things working, we can shave off time with additional optimizations later.